.circleci/config.yml

```diff
@@ -184,25 +184,29 @@ workflows:
       - build:
           name: Build
           requires:
-            - Install
+            - Lint
       - test:
           name: Test
           context: nodejs-install
           requires:
-            - Install
+            - Lint
```

Comment:
This should probably be
```yaml
- Lint
- Build
```
I think we can't run tests before Build happened! (I think we mount the built files from earlier)

Reply:
Not sure here though! Ignore me if it doesn't make sense haha

Reply:
Build and test have the same relationship after this PR as the old way (pictured) though don't they? 
![Screenshot 2020-11-18 at 17 13 25](https://user-images.githubusercontent.com/73909150/99563648-6089c280-29c1-11eb-942f-2a1fcb75f056.png)

Reply:
Or are you saying that the old way was wrong too? Sorry if Im being slow - long day with lots of CI 😅

Reply:
@ivanstanev I think this conversation got lost, can I bump it when you have a mo please?

Reply:
Test Jest right now (because it requires Node 10) does its own Install and doesn't need to wait for any other steps (similarly the Windows tests don't wait for anything because they cannot use the modules built in the Install step)! So I would guess the pipeline should look like this:

```
Test Windows
Test Jest Windows
Test Jest
Install -> Lint -> Test
                   Build
```

Reply:
Ah I see! Since test-jest does npm run lint anyway, I think it's probably not worth the effort changing this then, at least until we complete move to jest